### ISSUE TITLE
Enter key does not trigger <a> element onclick when JAWS is on

### ISSUE THREAD

**Issue by ctruong9** [FreedomScientific/VFO-standards-support#655]
### Summary

> Brief description of the issue. Please list any specific steps.

>  Can watch within the video.
1. tabbed over to the "Log Email on Send" button which implements the <a> element tag and has an onclick event handler.
2. Pressing "Enter" key does not work with JAWS screen reader on, but pressing "Enter" without JAWS on allows the screen reader to work properly.

> Example:
> 1. Go to http://www.google.com
> 2. Shift-Tab back to the About link

### Expected result

> What did you expect to happen?
Pressing "Enter" key with JAWS on should properly still trigger the button.

### Actual result

> What actually did happen?
Pressing "Enter" key with JAWS on does not trigger the button. A mouse click is required.

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number
JAWS version Windows 2022 2204.20 ILM

 
#### Operating System and version
Windows version 21H2 (OS Build 19044.1645)

#### Browser and version:
Outlook version 2204 (Build 15128.20178)

[JAWSKeyPressBug.mov.zip](https://github.com/FreedomScientific/VFO-standards-support/files/9151787/JAWSKeyPressBug.mov.zip)

**Comment by JAWS-test** [comment]
It is difficult to analyze the cause without more detailed information. In principle, however, JAWS intercepts all keystrokes and partially converts them, so that other event handlers must be used. For example, JAWS converts the keyboard event "Enter key" into a click event. JAWS can be configured so that the key events are not converted to click events.

**Comment by rappol** [comment]
Is there any update on this? i also have this kind of problem. I have a button inside an angular component that doesnt get triggered when using JAWS ...but without JAWS is working fine.
